sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `creature_template` ADD COLUMN `levelScalingDelta` SMALLINT(5) DEFAULT 0 NOT NULL AFTER `maxlevel`, CHANGE `flags_extra` `flags_extra` BIGINT(20) UNSIGNED DEFAULT 0 NOT NULL;
```

Comment:
It was another valable idea that i discussed with funjoker, but i felt it would be a duplicate of min & max levels